src/jsp/antlr4/SqlParser.g4

```diff
@@ -244,6 +244,10 @@ s_merge_insert_clause
       s_values_clause s_where_clause?
     ;
 
+s_truncate_statement
+    : TRUNCATE TABLE? s_table_ref CASCADE?
```

Comment:
To my surprise, it compiles and works well without defining CASCADE in PcsLexer.g4.
(I have tested a truncate statement having CASCADE, and it it compiles to Java code well)
However, it seems good to add CASCADE to PcsLexer.g4 at least for the keyword listing purpose.

Reply:
Thank you. I've added the CASCADE in PcsLexer.